### ISSUE TITLE
Can customization sync use automatic to get the new plugin for new devices?

### ISSUE THREAD

**Issue by lzism** [vrtmrz/obsidian-livesync#512]
## Abstract
Installation of new plug-ins on new devices is not possible with customization sync automatic.

## Expected behaviour
I have two devices, one device A is A commonly used device, and the addition and deletion of plug-ins were completed on A. Now there is A new device B that has no plug-in installed. I want to realize plug-in synchronization by opening customization sync automatic on the two devices, so as to synchronize the plug-in to be used from A and synchronize the modification of B to A

## Actually happened
Using the selection device works, but automatic does not

**Comment by vrtmrz** [comment]
Thank you for opening the topic!

You seem to have enabled `Customisation sync`. Is `Hidden file sync` also enabled? Automatic sync of `Customisation sync` means `Leave these files to Hidden file sync`.

I would like to integrate these two behaviours eventually. Sorry for the vague wording. I will improve it later. 

Would you mind if I ask you to check the behaviour with `Hidden file sync` enabled?

**Comment by lzism** [comment]
Apologies for taking so long to reply, I've now opted for a manual install on the new device, it's a bit of a hassle but the good thing is that the plugins don't change as often. I didn't choose to turn on Hidden file sync, I was under the impression that it syncs too many things I don't actually need, just a couple of plugins to safeguard the display on mobile. So what I envisioned was that with customization sync I could manage it on the PC, I've forgotten how it worked at the time, anyway it didn't work as I envisioned but looking at the presentation it should have!
